src/env.d/08-ca-certificates

```diff
@@ -0,0 +1,3 @@
+# Set the ssl certificates path for ruby.
+export SSL_CERT_DIR="${CREW_PREFIX}/etc/ssl/certs"
+export SSL_CERT_FILE"${CREW_PREFIX}/etc/ssl/certs/ca-certificates.crt
```

Comment:
Do we know if `CREW_PREFIX` has been set at this point?  The `08-...` may be a clue.

Reply:
yes, it has been set.

Reply:
Do we need to specify `export VARIABLE=something` in env.d files?

Doesn't `VARIABLE=something` suffice?

Reply:
thats not what `05-iconv` or `06-chromebrew` or `07-ld-library-path` do, so i'm not sure

Reply:
That's a bug. Fixing it here: https://github.com/chromebrew/crew-profile-base/pull/28

Reply:
ok, now you can merge with master and just take out the `export` parts of those lines...

Once that's done, we can merge this, and tag a new release?

Reply:
`CREW_PREFIX` is set in `/usr/local/etc/profile`, so we will always have that as an env variable.